adapters/repos/db/index.go

```diff
@@ -1987,6 +1988,28 @@ func (i *Index) dropShards(names []string) error {
 	return ec.ToError()
 }
 
+func (i *Index) dropCloudShards(ctx context.Context, cloud modulecapabilities.OffloadCloud, names []string, nodeId string) error {
+	i.shardTransferMutex.RLock()
+	defer i.shardTransferMutex.RUnlock()
+
+	ec := &errorcompounder.ErrorCompounder{}
```

Comment:
I think that `ec` will always return nil bc you just instantiate it here and then you do nothing with it. I think that this needs to removed and instead you should do `return eg.Wait()`

Reply:
good catch, actually should add the error from deletion to it

Reply:
will open a follow up PR

Reply:
https://github.com/weaviate/weaviate/pull/5303